src/components/table.js

```diff
@@ -593,17 +595,19 @@ export const SimpleTable = ({
         hover: useHover && { backgroundColor: colors.light(0.4) }
       }, [
         div({ style: { display: 'flex' } }, [
-          _.map(({ key, size }) => {
+          _.map(([colindex, { key, size }]) => {
             return div({
               key,
               role: 'cell',
+              'aria-rowindex': i + 2,
```

Comment:
Is the row index two because there's one row for the header followed by the data? If so, could we add that as a comment?
```suggestion
              'aria-rowindex': i + 2, // The first row of data is 2
```